packages/measure/src/measure-renders.tsx

```diff
@@ -69,6 +78,19 @@ export async function measureRender(ui: React.ReactElement, options?: MeasureOpt
     runResults.push({ duration, count });
   }
 
+  if (
+    jsonRepresentations.length >= minRecordToCompare &&
+    jsonRepresentations.every((json) => json === jsonRepresentations.at(0))
+  ) {
+    const testName = expect.getState().currentTestName;
+    logger.warn(`test "${testName}" has unnecessary renders. Please update your code to avoid unnecessary renders.`);
```

Comment:
Optimally we would like to surface this into output JSON file, and subsequently generated MD and screen reports.

Reply:
Sounds good! Basically we would like to have reporting like  `Count changes` etc.

Reply:
<img width="813" alt="Screenshot 2024-03-22 at 09 40 57" src="https://github.com/callstack/reassure/assets/40958989/003907e9-a66c-4daf-9d7a-184e7113749c">


I do save the result into output.json now and show the comparison like `Count Changes` shows